android/build.gradle

```diff
@@ -82,8 +82,10 @@ dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation 'androidx.annotation:annotation:1.3.0'
+    implementation 'androidx.fragment:fragment-ktx:1.6.2'
 
     api "com.revenuecat.purchases:purchases:$purchases_version"
+    api "com.revenuecat.purchases:purchases-ui:$purchases_version"
```

Comment:
Mentioned it over slack, but this is causing issues since it's API 24+ but this module is 19+.